src/handlers/deeplinks.ts

```diff
@@ -130,7 +130,6 @@ export default async function handleDeeplink(
               _action(asset);
             } else {
               dispatch(emitAssetRequest(address));
-              dispatch(emitChartsRequest(address));
```

Comment:
this should be replaced with a prefetchFn

Reply:
is that as asimple as `metadataClient.prefetchQuery({ ... })` im not seeing the type come thru for that

Reply:
we decided we don't need prefetch because it's being fetched when component mounts <200ms later